wp-autoupdates.php

```diff
@@ -731,15 +731,15 @@ function wp_autoupdates_debug_information( $info ) {
 				/* translators: %s: Latest plugin version number. */
 				$plugin_version_string       .= ' ' . sprintf( __( '(Latest version: %s)', 'wp-autoupdates' ), $plugin_updates[ $plugin_path ]->update->new_version );
 				/* translators: %s: Latest plugin version number. */
-				$plugin_version_string_debug .= ' ' . sprintf( __( '(latest version: %s)', 'wp-autoupdates' ), $plugin_updates[ $plugin_path ]->update->new_version );
+				$plugin_version_string_debug .= ' ' . sprintf( __( '(Latest version: %s)', 'wp-autoupdates' ), $plugin_updates[ $plugin_path ]->update->new_version );
```

Comment:
Why not completely collapse this into `$plugin_version_string`?

Reply:
Hi @jeffpaul,

$plugin_version_string and $plugin_version_string_debug both variable used for diffrent pepose.

$plugin_version_string in value and $plugin_version_string_debug use for debug

Reply:
Yep.
However @mukeshpanchal27 I think it would be better to use the lowercase version 😃 
Can you please update the patch accordingly?

Thank you!